qt/widgets/common/src/CatalogSearch.cpp

```diff
@@ -577,6 +576,16 @@ bool CatalogSearch::validateDates() {
   return ret;
 }
 
+void CatalogSearch::correctedToolTip(std::string text, QLabel *label) {
+#ifdef Q_OS_WIN
+  label->setToolTip(QString::fromStdString("<span style=\"color: black;\">" +
```

Comment:
Quick question. Do we know why we style the tooltip at all and not just leave it as text and let the theme decide?

Reply:
I assumed there was probably a reason behind the decision and so tried to keep it close to the original, when I was checking it appears that not setting a colour leads to it defaulting to red on black on ubuntu 18 at least.

Reply:
Okay. We suspect there is a bug in Qt4 surrounding this: https://github.com/mantidproject/mantid/blob/master/MantidPlot/src/ApplicationWindow.cpp#L11595. This interface is not heavily used and I think this is an adequate fix for it.